src/core/client/admin/routes/configure/components/Moderation.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { StatelessComponent } from "react";
+
+import { Typography } from "talk-ui/components";
+
+import Header from "./Header";
+
+const Moderation: StatelessComponent = ({ children }) => (
+  <div>
+    <Header>Perspective Toxic Comment Filter</Header>
+    <Typography>
+      Using the Perspective API, the Toxic Comment filter warns users when
```

Comment:
This seems to not be in a translation?

Reply:
This is just dummy stuff until we get to the configuration story.